starky/src/stark_verifier_circom.rs

```diff
@@ -1376,12 +1377,12 @@ template StarkVerifier() {{
             s,
             s,
             s,
-            SHIFT
-                .clone()
+            // TODO: It seemd to we only use F1G here and make sure we only need to F3G::from(constant) (don not need constant.clone()) when using the constant variables
+            F3G::from(SHIFT.clone())
```

Comment:
Yeap, here use Fr(aka.F1G). But we need to use `F3G::from(constant.clone())` as the ownership of `constant`(Here we mean `SHIFT`) will be moved into the `from` function.

Reply:
@eigmax   please reconfirm  this work too.

Reply:
yes, I agree.  Fr is ok here.

Reply:
@cyl19970726  Please remove the comments here.